frontend/src/components/WofInput.vue

```diff
@@ -2,8 +2,7 @@
   <div class="input">
     <div class="input__header">
       <h2 class="input__title">{{ name}}</h2>
-      <!-- This error will be replaced with WofSmallError component -->
-      <div class="temporary-error" v-if="error.length && !isFocused">{{ error }}</div>
+      <div class="temporary-error" v-if="error.length && !isFocused"><wof-small-error :text="error" :size="1" /></div>
```

Comment:
We don't need this div here. It was only temporary.
WofSmallError don't need to wrapped by this div.
Use this v-if directly on wof-small-error.
Set size to 1.5